src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_ICoalesceAssignmentOperation.cs

```diff
@@ -248,56 +248,53 @@ static void M()
         [Fact, CompilerTrait(CompilerFeature.Dataflow)]
         public void CoalesceAssignmentFlow_NullableValueTypeTarget()
         {
-            string source = @"
+            var comp = CreateCompilation(@"
 class C
 {
     static void M(int? i1, int i2)
     /*<bind>*/{
         i1 ??= i2;
     }/*</bind>*/
 }
-";
+");
+
+            VerifyOperationTreeAndDiagnosticsForTest<BlockSyntax>(comp, @"
+IBlockOperation (1 statements) (OperationKind.Block, Type: null) (Syntax: '{ ... }')
+  IExpressionStatementOperation (OperationKind.ExpressionStatement, Type: null) (Syntax: 'i1 ??= i2;')
+    Expression: 
+      ICoalesceAssignmentOperation (OperationKind.CoalesceAssignment, Type: System.Int32) (Syntax: 'i1 ??= i2')
+        Target: 
+          IParameterReferenceOperation: i1 (OperationKind.ParameterReference, Type: System.Int32?) (Syntax: 'i1')
+        Value: 
+          IParameterReferenceOperation: i2 (OperationKind.ParameterReference, Type: System.Int32) (Syntax: 'i2')
+", expectedDiagnostics: DiagnosticDescription.None);
+
             string expectedFlowGraph = @"
 Block[B0] - Entry
     Statements (0)
     Next (Regular) Block[B1]
         Entering: {R1}
-
 .locals {R1}
 {
-    CaptureIds: [0]
```

Comment:
>[0] [](start = 16, length = 3)

Just curious: what previously caused only one capture to be reported here? (we already had `IFlowCaptureOperation: 1` below)

Reply:
The capture was more locally scoped to just Region 2.

---
In reply to: [293125795](https://github.com/dotnet/roslyn/pull/36329#discussion_r293125795) [](ancestors = 293125795)